static/app/views/performance/trends/changedTransactions.tsx

```diff
@@ -545,6 +545,7 @@ const ChartContainer = styled('div')`
 `;
 
 const StyledHeaderTitleLegend = styled(HeaderTitleLegend)`
+  border-radius: ${p => p.theme.borderRadius};
```

Comment:
I feel like these 4 pixels are pretty exquisite, thank you @EvanPurkhiser.

Reply:
clutch pixels.

Reply:
👏